code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -278,10 +265,10 @@
 	if(isitem(target))
 		var/obj/item/I = target
 		if(!I.anchored)
-			to_chat(firer, "<span class='notice'>You pull [I] towards yourself.</span>")
-			add_attack_logs(src, I, "[src] pulled [I] towards them with a tentacle")
-			H.throw_mode_on()
-			I.throw_at(H, 10, 2)
+			to_chat(firer, "<span class='notice'>You grab [I] with your tentacle.</span>")
+			add_attack_logs(src, I, "[src] grabs [I] with a tentacle")
```

Comment:
`src` here should be `H` no? Else you're saying `"Tentacle grabs [I] with a tentacle"` as src is the projectile.

Reply:
SRC fixed